risc0/zkvm/src/session.rs

```diff
@@ -66,6 +66,9 @@ pub struct Session {
 
     /// The [ExitCode] of the session.
     pub exit_code: ExitCode,
+
+    /// The proof ID used for Bonsai execution
+    pub proof_id: Option<i64>,
```

Comment:
I would make this pub(crate) and consider hiding it from serialization with: `#[serde(skip_serializing)]`